tests/tests_pytorch/trainer/connectors/test_signal_connector.py

```diff
@@ -90,14 +90,37 @@ def test_auto_requeue_flag(auto_requeue):
 
         sigusr1_handlers = signal.getsignal(signal.SIGUSR1).signal_handlers
         assert len(sigusr1_handlers) == 1
-        assert sigusr1_handlers[0].__qualname__ == "SignalConnector.slurm_sigusr1_handler_fn"
+        assert sigusr1_handlers[0].__qualname__ == "SignalConnector.slurm_sigusr_handler_fn"
     else:
         assert signal.getsignal(signal.SIGTERM) is signal.SIG_DFL
         assert signal.getsignal(signal.SIGUSR1) is signal.SIG_DFL
 
     connector.teardown()
 
 
+@RunIf(skip_windows=True)
+@pytest.mark.parametrize("auto_requeue", (True, False))
+@pytest.mark.parametrize("requeue_signal", [(signal.SIGUSR1, signal.SIGUSR2, signal.SIGHUP)])
```

Comment:
I wouldn't duplicate the test above just for checking this parametrization. The difference is only a single line, why not merge the tests?

Reply:
Initially it was to make sure the default argument was working as expected but that's not as important with the updated interface from the above discussions